server/audit/tests.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework import status
+from rest_framework.test import APITestCase
+from rest_framework.test import APIClient
+from user_account.models import CustomUser
+from inventory_item.models import Item
+from .models import Audit
+
+
+class AuditTestCase(APITestCase):
+
+    def setUp(self):
+        self.client = APIClient()
+
+        # Create each type of user that could be making the registration request
+        self.system_admin = CustomUser.objects.create(
+            user_name='system_admin3',
+            email='system_admin3@email.com',
+            password='password2',
+            first_name='system2',
+            last_name='admin2',
+            role='SA',
+            is_active=True)
+
+        self.item_one = Item.objects.create(
+            _id=1,
+            Location='QC',
+            Plant=True,
+            Zone='B',
+            Aisle=3,
+            Part_Number='PART-1',
+            Part_Description='DummyPart',
+            Serial_Number='SN-4',
+            Condition='Servicable',
+            Category='SERIALIZED',
+            Owner='Me',
+            Criticality='3',
+            Average_Cost='$900',
+            Quantity=7,
+            Unit_of_Measure='EA'
+        )
+
+        self.item_two = Item.objects.create(
+            _id=2,
+            Location='QC',
+            Plant=True,
+            Zone='B',
+            Aisle=4,
+            Part_Number='PART-2',
+            Part_Description='DummyPart',
+            Serial_Number='SN-4',
+            Condition='Servicable',
+            Category='SERIALIZED',
+            Owner='Me',
+            Criticality='2',
+            Average_Cost='$400',
+            Quantity=4,
+            Unit_of_Measure='EA'
+        )
```

Comment:
This is actually not necessary i can just delete it because we are not asserting what is returned. But yeah fixture you be the right approach to be consistent.